docs/faq.md

```diff
@@ -7,10 +7,6 @@ top_graphic: 1
 
 [<- Back to Documentation List](/docs/)
 
-## Are certificates from Let’s Encrypt trusted by my browser?
```

Comment:
It's below: https://github.com/letsencrypt/website/pull/11/files#diff-956da29f025cba37fca9fe6b80ab5d48R14